src/coffea/nanoevents/schemas/nanoaod.py

```diff
@@ -206,6 +212,24 @@ def _build_collections(self, field_names, input_contents):
                     branch_forms["n" + name]
                 )
 
+        # Check the presence of the event_ids
+        missing_event_ids = [
+            event_id for event_id in self.event_ids if not hasattr(self, event_id)
+        ]
+        if missing_event_ids:
```

Comment:
```suggestion
        if len(missing_event_ids) > 0:
```
I think the length will need to be tested, as-implemented

Reply:
It is equivalent, maybe testing the length improves the clarity of the code but this is equivalent. (At least in the python version that I'm using, but I don't think it is a recent feature)

Reply:
You're right. I'll leave to others to comment, I prefer being explicit, but the functionality is the meaningful part, so I won't insist alone.

Reply:
"Explicit is better than implicit"?